### PR TITLE
FIX: Fer detect no resize

### PR DESCRIPTION
**Pull request by sihrc**

### CODE REVIEW

File: R/face_emotion.R [hunk 1]
```diff
@@ -41,10 +41,18 @@ face_emotion <- function(img, api_key = FALSE, cloud = FALSE, ...) {
...
 
-  img <- format_image(img, 64)
+  options <- list(...)
+  img <- format_image(img,
+      if(exists("detect", options) && options["detect"==TRUE]) 48 else FALSE
+  )
   make_request(img, 'fer', api_key, cloud, ...)
 }
 
+fn <- function(img, ...) {
```
Comment (madisonmay) [inline R/face_emotion.R]: I'm at a loss as to what's going on here. @sihrc ?

File: R/face_emotion.R [hunk 2]
```diff
@@ -41,10 +41,18 @@ face_emotion <- function(img, api_key = FALSE, cloud = FALSE, ...) {
     stop("No image for analysis provided!")
   }
 
-  img <- format_image(img, 64)
+  options <- list(...)
+  img <- format_image(img,
+      if(exists("detect", options) && options["detect"==TRUE]) 48 else FALSE
```
Comment (madisonmay) [inline R/face_emotion.R]: Shouldn't this be `options["detect"] == TRUE?`

Comment (sihrc) [inline R/face_emotion.R]: isn't it? or do you mean with the spaces?

Comment (sihrc) [inline R/face_emotion.R]: oh no. i see what you mean

**Pull request merged**